Update allow-members-to-send-as-or-send-on-behalf-of-group.md (#4340)
MicrosoftDocs/microsoft-365-docs: microsoft-365/solutions/allow-members-to-send-as-or-send-on-behalf-of-group.md

---

> title: "Allow Users to send as or send on behalf of a Group"

Reviewer: ```suggestion
title: "Allow users to send as or send on behalf of a group"
```

search.appverid:
- MET150
ms.assetid: 0ad41414-0cc6-4b97-90fb-06bec7bcf590
description: "Learn how to allow users to send email as a Microsoft 365 group or send email on behalf of a Microsoft 365 group."
---

# Allow members to send as or send on behalf of a group

> > [!Note]

Reviewer: ```suggestion
> [!NOTE]
```

> > Guests who are members of the group cannot be granted to send as or send on behalf of a group

Reviewer: ```suggestion
> Guests who are members of a Microsoft 365 group cannot be granted **Send as** or **Send on behalf** permissions for the group.
```

> description: "Learn how to allow members to send email as a Microsoft 365 group or send email on behalf of a Microsoft 365 group."

Reviewer: ```suggestion
description: "Learn how to allow users to send email as a Microsoft 365 group or send email on behalf of a Microsoft 365 group."
```

---

> # Allow Users to send as or send on behalf of a group

Reviewer: ```suggestion
# Allow users to send as or send on behalf of a group
```
